_posts/convars/2020-04-30-mom_hud_ghost_entpanels_enable_avatars.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: mom_entpanels_enable_avatars
+title: mom_hud_ghost_entpanels_enable_avatars
```

Comment:
This is `mom_hud_ghost_entpanels_avatars_enable`. Forgot to include that in the issue sorry!

Reply:
Resolved in [`2808c19` (#230)](https://github.com/momentum-mod/docs/pull/230/commits/2808c19000f132ec30c7ae65c39ec71f42462f11)